web/services/lib/filters/filterDefs.ts

```diff
@@ -364,6 +369,22 @@ export function filterUIToFilterLeafs(
   return filters
     .filter((filter) => filter.value !== "")
     .map((filter) => {
+      if (
+        filterMap &&
+        filterMap[filter.filterMapIdx].isCustomProperty &&
+        filterMap[filter.filterMapIdx].isCustomProperty === true
+      ) {
+        return {
+          request_response_versioned: {
+            properties: {
+              [filterMap[filter.filterMapIdx]?.column]: {
+                [filterMap[filter.filterMapIdx]?.operators[filter.operatorIdx]
+                  ?.value]: filter.value,
+              },
+            },
+          },
```

Comment:
btw i know i designed it like this initially, but if you have any ideas how to make this easier please suggest!